src/wikitextprocessor/parser.py

```diff
@@ -1919,8 +1919,7 @@ def tag_fn(ctx: "Wtp", token: str) -> None:
     else:
         m = re.match(r"</([-a-zA-Z0-9]+)\s*>", token)
         if m is None:
-            print("Could not match end tag token: {!r}".format(token))
-            assert False
+            raise Exception("Could not match end tag token: {!r}".format(token))
```

Comment:
Oh, I haven't even realized this; yeah, of course raising an Exception is better than assert False. I'll have to keep this in mind. Thanks for pointing this out.